### PR TITLE
Install CLI

### PR DESCRIPTION
**Pull request by doosuu**

### CODE REVIEW

**Comment by doosuu** [comment]
Should not be merged yet, there is an issue with this CLI version

**Comment by doosuu** [comment]
Can be reviewed now, bearer issue has been fixed.

File: Dockerfiles/Python/Dockerfile [hunk 1]
```diff
@@ -24,6 +24,12 @@ RUN apt-get update && apt-get upgrade -y && \
         libstdc++6 zlib1g locales sudo ncdu man-db strace gnupg2 \
         build-essential python3-dev
 
+# Install CLI
+ARG TARGETARCH
+COPY ./bin/velocitas-cli-$TARGETARCH /usr/bin/velocitas
+RUN chmod +x /usr/bin/velocitas
```
Comment (MP91) [inline Dockerfiles/Python/Dockerfile]: can't we do this before commiting? or is it not persistent?

File: Dockerfiles/Cpp/Dockerfile [hunk 2]
```diff
@@ -84,6 +84,12 @@ RUN apt-get --fix-missing update && apt-get upgrade -y && \
         update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-14 100 && \
         update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-14 100
 
+# Install CLI
+ARG TARGETARCH
+COPY ./bin/velocitas-cli-$TARGETARCH /usr/bin/velocitas
+RUN chmod +x /usr/bin/velocitas
```
Comment (MP91) [inline Dockerfiles/Cpp/Dockerfile]: can't we do this before commiting? or is it not persistent?

File: Dockerfiles/Python/Dockerfile [hunk 3]
```diff
@@ -51,3 +57,4 @@ RUN pip3 install -r requirements-app.txt
 RUN pip3 install -r requirements-tests.txt
 # Keep root as last user
 USER root
+WORKDIR /
```
Comment (MP91) [inline Dockerfiles/Python/Dockerfile:59]: is this needed? it was also working before or?

Comment (doosuu) [reply Dockerfiles/Python/Dockerfile:59]: Was working before when you enter the image interactively, you end up in /requirements which is not too great

File: Dockerfiles/Cpp/Dockerfile (hunk 2 above)
Comment (doosuu) [reply Dockerfiles/Cpp/Dockerfile]: Changed file permissions and removed this

File: Dockerfiles/Python/Dockerfile (hunk 1 above)
Comment (doosuu) [reply Dockerfiles/Python/Dockerfile]: Changed file permissions and removed this

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**